app/src/__tests__/Users.spec.ts

```diff
@@ -120,4 +130,54 @@ describe('Users.vue', () => {
 
     showToastSpy.mockRestore()
   })
+
+  it('filters data by a phrase', async () => {
+    await wrapper.setData({ users: usersData });
+    await wrapper.setData({ filteredUsers: usersData });
```

Comment:
``` suggestion
    await wrapper.setData({ 
       users: usersData,
       filteredUsers: usersData
 });
 
```

Reply:
Combined those into one wrapper.setData in the recent commit.

Reply:
Did you check all the tests?

Reply:
In the most recent commit there is just one users: usersData, because without online filtering logic is a little bit different

Reply:
Not sure if we are talking about the same thing, I meant using setData only once in [this](https://github.com/SolomiiaLomnytska22/solomiia-training/pull/26/files#diff-a8f90d4921851fc013ed020a3ca38337904b8e21e7b56613282ac26aef1d45e1R135-R137) case.

Reply:
See it now, thanks

Reply:
Fixed in the recent commit!